openreview/venue/venue.py

```diff
@@ -1137,7 +1137,7 @@ def ithenticate_create_and_upload_submission(self):
                     owner_first_name=first_name,
                     owner_last_name=last_name,
                     owner_email=owner_profile.get_preferred_email(),
-                    group_id=self.get_submission_id(),
+                    group_id=self.venue_id,
```

Comment:
Just out of curiosity, do `self.get_submission_id()` and `submission.id` return the same value?
Also, lines 1144 to 1156 is commented code. Can we remove it?

Reply:
`self.get_submission_id()` returns the submission invitation ID and `submission.id` returns the note ID of the submission. 

I've removed the commented code.

Reply:
I see. So confusing. OK, thanks!